modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -20,20 +20,19 @@ def index
         is_using_pagination = params[:page].present? || params[:per_page].present?
         resource.data = params[:include_image].present? ? fetch_and_include_images(resource.data) : resource.data
         resource = is_using_pagination ? resource.paginate(**pagination_params) : resource
-        render json: resource.data,
-               serializer: CollectionSerializer,
-               each_serializer: PrescriptionDetailsSerializer,
-               meta: resource.metadata
+
+        options = { meta: resource.metadata }
+        options[:links] = pagination_links(resource) if is_using_pagination
+        render json: MyHealth::V1::PrescriptionDetailsSerializer.new(resource.data, options)
```

Comment:
Can you also explain how the options param is doing here? Is it going to change the structure of the object? For example, add `{ options: { ...meta info } }` instead of including the meta info in the root of of the object?

Reply:
This includes the meta information at the root level. 

My intent was to keep the structure and values of the responses exactly the same. So if you do see any discrepancies please let me know.

Reply:
I checked and it looks the same.